src/components/TextWithTooltip.tsx

```diff
@@ -11,12 +11,14 @@ export interface TextWithTooltipProps extends TypographyProps {
   iconSize?: number;
   // eslint-disable-next-line
   children?: ReactElement<any, string | JSXElementConstructor<any>>;
+  color?: string;
 }
 
 export const TextWithTooltip = ({
   text,
   icon,
   iconSize = 14,
+  color = 'text.muted',
```

Comment:
Should this perhaps just consistently always be like that? I mean instead of making it a prop

Reply:
Feels weird to me that in this case it should be error color while essentially it's just an info like in all other cases.

Reply:
Reverted it back to the original. I thought it would make it stand out more but I agree it looked kinda weird